presets/4.3/vtx/diatone_mamba_ultra_1000.txt

```diff
@@ -0,0 +1,38 @@
+#$ TITLE: Diatone Mamba Ultra 1000 VTX Tables
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS:  vtx, vtx table, diatone, mamaba, ultra 1000
+#$ AUTHOR: NemesisXB
+#$ DESCRIPTION: VTX tables for the Diatone Mamba Ultra 1000.
+#$ DESCRIPTION: Select the the HAM option if you have a HAM license or the NO HAM if you do not.
+#$ DESCRIPTION: EU users select the NO HAM option.
+#$ DISCLAIMER:  Only select one option. All previous VTX Table settings will be reset.
+#$ INCLUDE_DISCLAIMER: misc/disclaimer/en/vtxtable.txt
+#$ DISCUSSION:  https://github.com/betaflight/firmware-presets/pull/107
+
+#$ INCLUDE: presets/4.3/vtx/defaults_vtx_tables.txt
+
+# Configure common vtxtable settings
+vtxtable bands 5
+vtxtable channels 8
+vtxtable powerlevels 5
+vtxtable powervalues 25 100 200 400 600
+vtxtable powerlabels 25 200 400 800 1W
```

Comment:
I am curious why power levels do not match with the power labels.... And i see it is like that in the JSON file too 0_o
Do you have this VTX? Could you check if that's SA 2.1 or SA 2.0?
Configurator 4.3 shows SA version on vtx tab on the right side.

Reply:
I have a friend with one, but he is not running 4.3. I know it is Tramp, does it also have version number?

The manual also shows the same values, so it is not just in the json file.
![image](https://user-images.githubusercontent.com/485436/145712935-5ddde17e-13d9-4fd6-9306-1d7d1e71bdee.png)

Reply:
We can merge this no problem as it matches with the official tutorial.
Maybe it's too much to ask, could you email to Diatone, say that you work on Betaflight Preset and you need to clarify why the power levels don't match with the labels?

For HDZero it was kinda a similar problem we found, and @sugaarK emailed to them and they found a bug in the firmware thanks to us LOL

Reply:
> We can merge this no problem as it matches with the official tutorial. Maybe it's too much to ask, could you email to Diatone, say that you work on Betaflight Preset and you need to clarify why the power levels don't match with the labels?
> 
> For HDZero it was kinda a similar problem we found, and @sugaarK emailed to them and they found a bug in the firmware thanks to us LOL

Yeah sure. Can do.

Reply:
@limonspb, Diatone replied pretty quickly. but I don't understand it anyway.

This is their response:
The power setting and the value of the image transmission do not necessarily correspond. It depends on the program setting. The image transmission manual is directly marked. Can be seen.

-----------------
"Image transmission power setting value" and image transmission (hardware) in BF, subject to the image transmission (hardware) program

For example: the power in the image transmission program is: 400, 600, 800, 1000
Should be entered in BF: 400, 600, 800, 1000

If you enter other values in BF, the image transmission (400, 600, 800, 1000) will prevail

Reply:
it is hard to understand yeah :)
I guess we should adjust values and labels so they match with the true VTX output power and be done with it.
Could you please also add into title something like Diatone Mamba Ultra 1000 VTX Tables (SA2.0)
And SmartAudio 2.0 in the description?
In case they release a new version of the vtx that is 2.1 - that happens sometimes. At least people will see that this vtx is 2.0 only.
And i guess we are done :)

Reply:
@limonspb, why SA2.0? This VTX uses TRAMP.